packages/webpack-inline-constant-exports-plugin/index.js

```diff
@@ -113,8 +113,9 @@ class InlineConstantExportsPlugin {
 						const usesConstantDependencies = new Map(); // Module -> Boolean
 						const importSpecifierDependencies = []; // [ ImportSpecifierDependency ]
 
-						for ( const dep of module.dependencies ) {
-							if ( ! this.isConstantsModule( dep.module ) ) {
+						for ( const dep of module.dependencies.values() ) {
+							const dependencyModule = compilation.moduleGraph.getModule( dep );
```

Comment:
https://github.com/webpack/webpack/blob/4ba6d2ea67e9046cbba83f588989b3cadff7d911/lib/Dependency.js#L217

`dep.module` is gone, webpack suggest to use the module graph